CHANGES/10851.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix pytest plugin to not use deprecated asyncio policy APIs.
```

Comment:
Also, I believe we can use `` :py:mod:`asyncio` `` to link the corresponding doc here.